<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyCarts</string>
  </property>
  <widget class="QPushButton" name="main_p">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Основной состав</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_person">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="all_p">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Общий состав</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="info_text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choise">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="num_text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="name_text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="city_text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="network_text">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_city">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Город</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_network">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Социальные сети</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Остальная информмация</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="age_text">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Возраст</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_main">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>330</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить в основной состав</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
